packages/polymorphic/src/Example/Polymorphic.example.tsx

```diff
@@ -48,16 +52,37 @@ export const ExamplePolymorphicWithRef = Polymorphic<ExampleProps>(
   'ExamplePolymorphicWithRef',
 );
 
-export const ExampleInferred = Polymorphic<
-  InferredPolymorphicProps<ExampleProps>
->(({ as, title, ...rest }) => {
+/**
+ * @example
+ */
+export const ExampleInferred = InferredPolymorphic<ExampleProps>(
```

Comment:
does this need defaults? What happens if there are none?

Reply:
This is an example for usage without a default. 

`ExampleInferredDefaultButton` is the example for usage with a default